agent/bench-scripts/driver/linpack

```diff
@@ -27,8 +27,8 @@
 #     line 6:     2     2     2
 #     line 7:     4     4     4
 #
-# This script provides optional input parameters for lines 1, 2, 4 - 7, and
-# then calculates the number of tests to be run from those inputs to fill in
+# This script offers input parameters for lines 1, 2, 4 - 7, with default values
+# and then calculates the number of tests to be run from those inputs to fill in
 # line 3 of the LINPACK input file.
```

Comment:
Perhaps something like, "This script provides optional command line switches to override the default values for lines 1, 2, 4-7; the script calculates...".